phys/module_mp_gsfcgce_4ice_nuwrf.F

```diff
@@ -676,18 +676,18 @@ SUBROUTINE fall_flux ( dt, ql, qr, qi, qs, qg, qh, p,       &
   REAL                          :: t_del_tv, del_tv, flux, fluxin, fluxout
   LOGICAL                       :: notlast
 
-  if (itimestep.eq.1) then
-     write(6, *) 'GSFCGCE 4ice scheme inside fall_flux'
-     write(6, *) 'improve = ', improve
-    
-     write(6, *) 'ims=', ims, '  ime=', ime
-     write(6, *) 'jms=', jms, '  jme=', jme
-     write(6, *) 'kms=', kms, '  kme=', kme
-     write(6, *) 'its=', its, '  ite=', ite
-     write(6, *) 'jts=', jts, '  jte=', jte
-     write(6, *) 'kts=', kts, '  kte=', kte
-     write(6, *) 'dt=', dt
-  endif 
+! if (itimestep.eq.1) then
```

Comment:
Wei,
I see two place where _I_ should have deleted print statements, and you commented them out. I would prefer that you just delete them.